bootstrap_2.3.1/css/bootstrap.css

```diff
@@ -223,10 +223,11 @@ textarea {
 
 body {
   margin: 0;
-  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
-  font-size: 14px;
-  line-height: 20px;
-  color: #333333;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 13px;
+  line-height: 2.4;
+  letter-spacing: 0.24;  
+  color: #414141;
```

Comment:
I see that you changed `primary_color` to this below... can you then include the mixin here instead of the hex value?